index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="icon" href="./images/apple-touch-icon-57x57.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="./index.css">
+    <title>Intuit Accounts - Sign Up</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="nav-right-column">
+                <div class="description">
+                    <div class="fade">Already have an Intuit account?</div>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <button type="button">
+                    <i class="material-icons icons" style="font-size: 14px;">lock</i>
+                    Sign In
+                </button>
+            </div>
+            <div class="nav-left-column"></div>
+        </nav>
+    </header>
+    <main>
+        <aside class="col-1"></aside>
+        <section class="container">
+            <div class="container-header">
+                <div class="banner"></div>
+                <ul class="menu-sprite">
+                    <li class="icon logo-1"></li>
+                    <li class="sprint mint"></li>
```

Comment:
This should not be a background image, but a text with mint as a content. The same goes for the Quickbooks and Turbotax below (lines 37 and 39)